src/lib/vm/vm.js

```diff
@@ -1612,6 +1612,21 @@ export default class VM {
     };
 
     const Near = {
+      signMessage: (...args) => {
+        if (args.length < 3) {
```

Comment:
Let's add similar API to call, where if the only 1 argument is given and it's an object, then it passes it directly